SAGESharp/SLB/Common.cs

```diff
@@ -446,23 +448,78 @@ private void SetByteValue(byte bytePosition, byte value)
         }
     }
 
-    internal sealed class Point3D : IEquatable<Point3D>
+    public sealed class Point3D : IEquatable<Point3D>, INotifyPropertyChanged
     {
+        private float _x;
```

Comment:
In other classes we haven't added underscore to private variables, just for consistency we may want to remove them (or add them in the other classes).

Reply:
Will do, good catch